observability/otel/otel_rn/otel-rn-3-2.adoc

```diff
@@ -0,0 +1,87 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="otel-rn-3-2"]
+= Release notes for {OTELName} 3.2
+:context: otel-rn-3-2
+
+toc::[]
+
+include::modules/otel-product-overview.adoc[leveloffset=+1]
+
+The {OTELName} is provided through the {OTELOperator}.
+
+////
+[id="otel-rn_3-2_cves"]
+== CVEs
+
+This release fixes link:https://access.redhat.com/security/cve/cve-202?-?????[CVE-202?-?????].
+////
+
+[id="otel-rn_3-2_technology-preview-features"]
+== Technology Preview features
+
+This update introduces the following Technology Preview features:
+
+// https://issues.redhat.com/browse/TRACING-3802 > https://docs.openshift.com/container-platform/4.14/observability/otel/otel_rn/otel-rn-past-releases.html#otel-rn_3-1_technology-preview-features
+////
+* The Target Allocator is an optional component of the OpenTelemetry Operator that shards Prometheus Receiver's scrape targets across the deployed fleet of OpenTelemetry Collector instances. The Target Allocator provides integration with the Prometheus `PodMonitor` and `ServiceMonitor` custom resources.
+////
+
+* Host Metrics Receiver
+* OIDC Auth Extension
+* Kubernetes Cluster Receiver
+* Kubernetes Events Receiver
+* Kubernetes Objects Receiver
+* Load-Balancing Exporter
+* Kubelet Stats Receiver
+* Cumulative to Delta Processor
+* Forward Connector
+* Journald Receiver
+* Filelog Receiver
+* File Storage Extension
+
+:FeatureName: Each of these features
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="otel-rn_3-2_new-features-and-enhancements"]
+== New features and enhancements
+
+This update introduces the following enhancement:
+
+* {OTELName} 3.2 is based on the open source link:https://opentelemetry.io/[OpenTelemetry] release 0.99.0.
+
+////
+[id="otel-rn_3-2_jaeger-release-notes_deprecated-functionality"]
+== Deprecated functionality
+
+In the {OTELName} 3.2, ???
+////
+
+////
+[id="otel-rn_3-2_removal-notice"]
+== Removal notice
+
+In the {OTELName} 3.2, the FEATURE has been removed. Bug fixes and support are provided only through the end of the 3.? lifecycle. As an alternative to the FEATURE for USE CASE, you can use the ALTERNATIVE instead.
+////
+
+////
+[id="otel-rn_3-2_bug-fixes"]
+== Bug fixes
```

Comment:
This OTEL operator bug is fixed. https://issues.redhat.com/browse/TRACING-3761

Reply:
This does not seem to be reported by customers either, but I would include it as it is quite visible